contrib/fmhash/fmhash.c

```diff
@@ -0,0 +1,382 @@
+/*
+ * Copyright (c) 2018-2020, Harshvardhan Shrivastava
```

Comment:
Copyright 2020?

Reply:
Wrong copy paste

Reply:
I fixed it in https://github.com/rsyslog/rsyslog/pull/2722